tcms/core/utils/__init__.py

```diff
@@ -1,13 +1,14 @@
 # -*- coding: utf-8 -*-
+#  pylint: disable=too-few-public-methods
 
 import re
 from django.conf import settings
 
-numeric_re = re.compile(r'^\d+$')
+NUMERIC_RE = re.compile(r'^\d+$')
 
 
-def is_int(s):
-    return numeric_re.match(s) is not None
+def is_int(string):
+    return NUMERIC_RE.match(string) is not None
```

Comment:
Is this constant used elsewhere ?

Also is is_int used in many places ? Sounds like something that can be removed.

Reply:
`NUMERIC_RE` is not used anywhere else, and `is_int` is used just in one place. I may think of a way to remove them.